regression/scripts/fuel_tracking_mission/fuel_tracking_mission.py

```diff
@@ -0,0 +1,1041 @@
+    # is insufficient for the desired descent speed and rate.
+    analyses.landing.aerodynamics.settings.spoiler_drag_increment = 0.00
+
+    segment.altitude_end = 4.0   * Units.km
+    segment.air_speed    = 170.0 * Units['m/s']
+    segment.descent_rate = 5.0   * Units['m/s']
+    
+    # Add to mission
+    mission.append_segment(segment)
+
+    # ------------------------------------------------------------------
+    #   Fourth Descent Segment: Constant Speed, Constant Rate
+    # ------------------------------------------------------------------
+
+    segment = Segments.Descent.Constant_Speed_Constant_Rate(base_segment)
+    segment.tag = "descent_4"
+
+    segment.analyses.extend( analyses.landing )
+    analyses.landing.aerodynamics.settings.spoiler_drag_increment = 0.00
+
+    segment.altitude_end = 2.0   * Units.km
+    segment.air_speed    = 150.0 * Units['m/s']
+    segment.descent_rate = 5.0   * Units['m/s']
+
+    # Add to mission
+    mission.append_segment(segment)
+
+    # ------------------------------------------------------------------
+    #   Fifth Descent Segment: Constant Speed, Constant Rate
+    # ------------------------------------------------------------------
+
+    segment = Segments.Descent.Constant_Speed_Constant_Rate(base_segment)
+    segment.tag = "descent_5"
+
+    segment.analyses.extend( analyses.landing )
+    analyses.landing.aerodynamics.settings.spoiler_drag_increment = 0.00
+
+    segment.altitude_end = 0.0   * Units.km
+    segment.air_speed    = 145.0 * Units['m/s']
+    segment.descent_rate = 3.0   * Units['m/s']
+    
+
+    # Append to mission
+    mission.append_segment(segment)
+
+    # ------------------------------------------------------------------
+    #   Mission definition complete    
+    # ------------------------------------------------------------------
+
+    return mission
+
+def missions_setup(base_mission):
+    """This allows multiple missions to be incorporated if desired, but only one is used here."""
+
+    # Setup the mission container
+    missions = SUAVE.Analyses.Mission.Mission.Container()
+
+    # ------------------------------------------------------------------
+    #   Base Mission
+    # ------------------------------------------------------------------
+
+    # Only one mission (the base mission) is defined in this case
+    missions.base = base_mission
+
+    return missions  
+
+def check_results(results):
+    #checking the weight values at the final segement 
+
+    weights = results.segments[-1].conditions.weights
+
+
+    #stored weight values to check against results
+    total_mass                   = np.array([[62696.17099197],[62694.1442812 ],[62688.14554268],[62678.41600768],[62665.3481715],[62649.47228568],[62631.43866482],[62611.9962415 ],[62591.96729459],[62572.21788392],[62553.62354735],[62537.03040895],[62523.21294419],[62512.83097265],[62506.38959385],[62504.20634677]])
```

Comment:
You should just include this entire regression to an existing regression script.